cypress/integration/HeaderBar/the_headerbar_contains_a_profile_menu/the_user_name_and_email_are_displayed.js

```diff
@@ -2,17 +2,18 @@ import '../common/index'
 import { Then } from 'cypress-cucumber-preprocessor/steps'
 
 Then('contains the user name', () => {
-    cy.get('@meFixture').then(({ name }) => {
-        cy.get('[data-test="headerbar-profile-username"]').then($name => {
+    cy.getAll('@meFixture', '[data-test="headerbar-profile-username"]').should(
```

Comment:
There's a `cy.all` helper that could be used as well:

```js
cy.all(
  () => cy.get('@meFixture'),
  () => cy.get('[data-test="headerbar-profile-username"]')
).should(([{ name }, $name]) => expect($name.text()).to.equal(name)
```

To me this is easier to parse visually

Reply:
Yeah I agree, I also think all uses of getAll could be moved to `all`

Reply:
Noted in this issue: https://github.com/dhis2/ui/issues/86#issuecomment-632147319